src/Microsoft.TemplateEngine.Cli/Commands/CompletionItemComparer.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.CommandLine.Completions;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.TemplateEngine.Cli.Commands
+{
+    internal class CompletionItemComparer : IEqualityComparer<CompletionItem>, IComparer<CompletionItem>
+    {
+        internal static CompletionItemComparer Instance { get; } = new CompletionItemComparer();
+
+        public int Compare(CompletionItem? x, CompletionItem? y)
+        {
+            if (x == null && y == null)
+            {
+                return 0;
+            }
+            if (x == null)
+            {
+                return -1;
+            }
+            if (y == null)
+            {
+                return 1;
+            }
+            return string.Compare(x.Label, y.Label);
```

Comment:
`CompletionItem` actually has these exact equality semantics already so this comparer should not be necessary.

Reply:
Removed custom comparer, added test instead to detect change in behavior.